readme.md

```diff
@@ -192,7 +185,25 @@ A couple of examples:
 
 For additional guidance and important information about certificates, please refer to [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-changes-are-coming-and-why-you-should-care/ba-p/1658456) from the security team.
 
-## Long-Term Support (LTS)
+## Support
+
+The Azure IoT Hub Device Client supported releases is outlined in the following table.
+
+Refer to the [Azure IoT Device SDK lifecycle and support](https://learn.microsoft.com/azure/iot/iot-device-sdks-lifecycle-and-support) for details on the different supported stages.
+
+| Release | Category | End-of-life |
+|-|-|-|
+| 1.41.3 | Active | - |
+| 1.36.8 | Deprecated (LTS) | 2024-03-28 |
+| 2.0.0-preview004 | Preview | 2023-06-13
```

Comment:
Newer preview out and I'd say previews don't really have an end-of-life as they aren't officially supported anyway.

```suggestion
| 2.0.0-preview005 | Preview | N/A |
```

Reply:
yeah been a bit back and forward on this. We do have some level on support for previews via github issues, but no production level support which I think you are talking about. But if we leave this blank, or N/A, it sounds to me like we are supporting it forever?

Reply:
If we want an actual date for previews, then I'd pick a shorter timespan, like 3-6 months.

Reply:
I agree, I was thinking 3 months. I just updated the date to be 3 months from the preview5 release